src/main/scala/io/razem/influxdbclient/RetentionPolicyManagement.scala

```diff
@@ -5,10 +5,14 @@ import scala.concurrent.Future
 protected[influxdbclient] trait RetentionPolicyManagement { self: Database =>
   def createRetentionPolicy(name: String, duration: String, replication: Int, default: Boolean): Future[QueryResult] = {
     var stringBuilder = new StringBuilder()
-      .append("CREATE RETENTION POLICY \"").append(name)
-      .append("\" ON \"").append(databaseName)
-      .append("\" DURATION ").append(duration)
-      .append(" REPLICATION ").append(replication)
+      .append("CREATE RETENTION POLICY \"")
```

Comment:
This is why I do not like autoformating sometimes. The way it was before made more sense. Can we disable autoformating for methods or code blocks?